packages/oui-dropdown/src/dropdown-trigger.directive.js

```diff
@@ -33,8 +33,8 @@ export default $compile => {
             triggerElement.attr("id", ctrl.id);
 
             triggerElement.on("click", () => ctrl.onTriggerClick());
-            triggerElement.on("keydown", evt => ctrl.triggerKeyHandler(evt));
             triggerElement.on("blur", evt => ctrl.triggerBlurHandler(evt));
+            $document.on("keydown", evt => ctrl.triggerKeyHandler(evt));
```

Comment:
(optional) Could it be registered only when the dropdown is expanded ? Because here if we have multiple instance of the trigger each time keydown will pass through those handler even if no dropdown is shown, unless I missed something.